<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialogBase</class>
 <widget class="QDialog" name="LoginDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="lblMessage">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_4">
     <item>
      <widget class="QLabel" name="lblProfile">
       <property name="text">
        <string>Profile:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbProfile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRename">
       <property name="text">
        <string>&amp;Rename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelNew">
     <property name="text">
      <string>New profile name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="e_newName"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxPasswords">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Passwords:</string>
     </property>
     <layout class="QVBoxLayout" name="_2">
      <item>
       <layout class="QVBoxLayout" name="inputsLayout"/>
      </item>
      <item>
       <widget class="QCheckBox" name="chkSave">
        <property name="text">
         <string>&amp;Save passwords</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkNoShow">
     <property name="text">
      <string>&amp;Do not show this window on startup</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_3">
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkNoShow</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>cmbProfile</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>chkSave</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>LoginDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>LoginDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbProfile</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>LoginDialogBase</receiver>
   <slot>profileChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSave</sender>
   <signal>clicked(bool)</signal>
   <receiver>LoginDialogBase</receiver>
   <slot>saveToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRename</sender>
   <signal>clicked()</signal>
   <receiver>LoginDialogBase</receiver>
   <slot>profileRename()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDelete</sender>
   <signal>clicked()</signal>
   <receiver>LoginDialogBase</receiver>
   <slot>profileDelete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>e_newName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LoginDialogBase</receiver>
   <slot>newNameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>profileChanged(int)</slot>
  <slot>saveToggled(bool)</slot>
  <slot>profileRename()</slot>
  <slot>profileDelete()</slot>
  <slot>newNameChanged(QString)</slot>
 </slots>
</ui>
